lib/Utils/Paths.cpp

```diff
@@ -0,0 +1,229 @@
+//--------------------------------------------------------------------*- C++ -*-
+// CLING - the C++ LLVM-based InterpreterG :)
+// author:  Jermaine Dupris <support@greyangle.com>
```

Comment:
On 16/08/16 11:18, marsupial wrote:
>
> In lib/Utils/Paths.cpp 
> <https://github.com/root-mirror/cling/pull/26#discussion_r74901399>:
>
> > @@ -0,0 +1,229 @@
> > +//--------------------------------------------------------------------*- C++ -*-
> > +// CLING - the C++ LLVM-based InterpreterG :)
> > +// author:  Jermaine Dupris <support@greyangle.com>
>
> Yes, no legal issues.
>
Please, remove authors. They will be mentioned in the CREDITS.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/root-mirror/cling/pull/26/files/fb0e73d441fb1f128482dc90496ad43c1d38711c#r74901399>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AGNuU2u0Z_y_mze3U_T4HzJmAGvua_aVks5qgYB6gaJpZM4JS_ds>.
>